.github/copilot-instructions.md

```diff
@@ -1,92 +1,125 @@
-- Custom helpers in `app/lib`.
-
-## Migrations
-- Data migrations must be included as a rake task outside of rails database migrations.
-- Index updates must always be included in a migration file by itself.
-- Index updates must be performed with the concurrently algorithm and outside of the DDL transaction to avoid locking.
-
-## Code Quality
-- Runs RuboCop for linting.
-- Document complex logic with comments.
-
-## Security
-- Don't log anything that could contain PII or sensitive data, including an entire response_body and `user.icn`.
-- Never commit secrets or keys.
-
-## Adding Features
-- Add new controllers for new resources.
-- Write tests for all new code.
-
-# Tips for Copilot
-- Prefer existing patterns and structure.
-- Reuse helpers and services when possible.
-- Write clear, concise code.
-- If asked to create an issue, create it in the department-of-veterans-affairs/va.gov-team repository.
-
-# Code Review Guidelines
-When performing a code review, ensure the code follows best practices. Additionally, pay close attention to these specific guidelines:
-
-## Ruby shorthand syntax
-- Always enforce Ruby shorthand syntax.
-- If a local variable is defined, using shorthand syntax like `{ exclude: }` is valid and correct.
-- Do **not** suggest that the key is missing a value.
-- Do **not** suggest changing it to `{ exclude: exclude }`.
-- Do **not** flag this syntax as an error, incomplete, or unclear.
-
-## Flipper usage in tests
-- Avoid enabling or disabling Flipper features in tests.
-- **Never** use `Flipper.enable` or `Flipper.disable` in tests.
-- Always stub Flipper like this:
-  ```ruby
-  allow(Flipper).to receive(:enabled?).with(:feature_flag).and_return(true)
-  ```
-  - Use this exact pattern inline in the example.
-
-## Active Record index migrations
-### Isolate index changes
-- If a migration includes `add_index` or `remove_index`, it must only include index changes.
-- Do **not** combine index changes with other table modifications in the same migration.
-
-### Avoid locking
-- A migration that includes `add_index` or `remove_index` must also:
-  - Use `algorithm: :concurrently`
-  - Include `disable_ddl_transaction!`
-
-This prevents table locking during deployment.
+# Copilot Instructions for `vets-api`
+
+## Repository Context
+`vets-api` is a Ruby on Rails API serving veterans via VA.gov. Large codebase (400K+ lines) with modules for appeals, claims, healthcare, and benefits processing.
+
+**Key External Services:** BGS, MVI, Lighthouse APIs, EVSS (being deprecated)
+**Architecture:** Rails engines in `modules/`, background jobs via Sidekiq Enterprise
+
+## For Copilot Chat - Development Help
+
+### Quick Commands
+- **Test**: `bundle exec rspec spec/` or `make spec`
+- **Lint**: `bundle exec rubocop` (handled by CI, don't suggest fixes)
+- **Server**: `foreman start -m all=1`
```

Comment:
I don't think I've ever used this command... Is this something you use? I use `bundle exec rails server`.